src/main/java/com/incentives/piggyback/offers/serviceImpl/OfferServiceImpl.java

```diff
@@ -109,4 +103,34 @@ public String sendNotification(BroadcastRequest broadcastRequest) {
 		return response.getBody().getData();
 	}
 
+	@Override
+	public List<UserData> getUsersWithInterest(List<Long> users, List<String> interests) {
```

Comment:
A partnerOrder will have having a single partnerOrderCategory. So getUsersWithInterest should be accepting a single interest field rather than a list of interests.

Reply:
reduced to one